Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -169,11 +206,59 @@ equation
           -52}}, color={0,0,127}));
   annotation (Documentation(info="<html>
 <p>
-This blocks implements a AMIGO PID tuning method and the control process is approximated with a first order delay process.
+This block implements an AMIGO PID tuning method and the control process is approximated with a first-order delay (FOD) process.
+</p>
+<p>
+Specifically, this AMIGO PID tuning method is based on asymmetric relay feedback tests, providing process excitation 
+at the frequency intervals relevant for PID control;
+</p>
+<p>
+Based on the response of the control process to the excitation, the parameters of a FODs, time delay and time constant, are calculated;
+</p>
+<p>
+Finally, the parameters of a PID controller are determined according to the parameters of the FOD.
+</p>
+<p>
+This block consists of a relay controller and a PID controller with input gains.
 </p>
 <p>
 Noted that this block can only support a PI controller or a PID controller.
 </p>
+<h4>Breif guidance</h4>
+<p>
+This block is built upon PIDWithInputGains<a href=\"modelica://Buildings.Controls.OBC.Utilities.PIDWithInputGains\">
+Buildings.Controls.OBC.Utilities.PIDWithInputGains</a> and its connections are the same as those of the latter.
+</p>
+<p>
+All the parameters of PIDWithInputGains are also used by this block, except the controller gains
+<i>k</i>, <i>T<sub>i</sub></i>, and <i>T<sub>d</sub></i>, which are replaced with the initial values of those gains.
+</p>
+<p>
+This block provides four additional parameters, <i>yHig</i>, <i>yLow</i>, <i>deaBan</i>, and <i>yRef</i>.
+</p>
+<p>
+Note that the default values of those four parameters assume the control output is normalized.
```

Comment:
How is the output normalized? Is it by setting the parameter <code>r</code>, or parameter <code>yRef</code>? Or should it be normalized externally?

Reply:
The normalized output means that the output of the PID controller is limited from 0 to 1. I revised the doc to avoid confusions.